samples/MachineKey/MachineKeyFramework/packages.config

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNetCore.Cryptography.Internal" version="2.2.0" targetFramework="net472" />
```

Comment:
Use 2.1, not 2.2 for all of these.

Reply:
I just installed the package and it added these. Should we be delisting the 2.2.0 packages from nuget so people use the 2.1.0 versions?

Reply:
wow I don't miss packages.config :( I've updated them to 2.1

Reply:
2.2 is already flagged as deprecated, nuget _shouldn't_ recommend them.